stock_no_negative/tests/test_stock_no_negative.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2015-2016 Akretion (http://www.akretion.com) - Alexis de Lattre
+# Copyright 2016 ForgeFlow (http://www.forgeflow.com)
+# Copyright 2016 Serpent Consulting Services (<http://www.serpentcs.com>)
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestStockNoNegative(TransactionCase):
+    at_install = False
+    post_install = True
+
+    def setUp(self):
```

Comment:
I would have done in a class method.

Reply:
If I understood correctly, these changes should be correct now, could you please confirm it? thank you

Reply:
I think there's no need of the post_install flag for being correctly tested in this case, as it was without it on 14.0.

Reply:
Ok! :+1: